frontend/src/views/Create.tsx

```diff
@@ -35,10 +35,13 @@ function CreateGamePage() {
 
   // Render the "Enter nickname" state.
   return (
-    <EnterNicknamePage
-      enterNicknameHeaderText="Enter a nickname to create the game!"
-      enterNicknameAction={createJoinLobby}
-    />
+    <>
+      <EnterNicknamePage
+        enterNicknameHeaderText="Enter your name to create the room!"
+        enterNicknameAction={createJoinLobby}
+      />
+      <p>As the host, you will be a spectator by default.</p>
```

Comment:
No problem if not on this, this could be a  future idea to look into as well since lots of changes are on this PR - thoughts @alankbi?

Reply:
Missed this comment -

Imo, I still think one of the better options would be to add some text to the lobby page to make it very clear what's going on, maybe something like this: 

![Text that says you're a spectator](https://user-images.githubusercontent.com/24768574/121825530-d9a37c80-cc67-11eb-88d5-5c4b983c45c9.png)

This was just a quick inspect element mockup, and idk what your thoughts are on how this looks (since I can see how it might be a little intrusive in the current spot). Any thoughts on this or other solutions?

Reply:
Hmm, I'd rather find some way to highlight the existing settings change options that exist, like your own player icon flashing for a bit to get the user to hover over the own card and know how to change their spectator settings; as you mentioned, I feel this is a little bit too intrusive in that spot, especially if players are already aware of the settings they prefer.

Reply:
No solution is immediately critical for this PR though imo - after all, I think most teachers _will_ want to be spectators in their games.

Reply:
Yeah I guess we can leave this till after getting feedback then, see what teachers think would make things the most intuitive. Another option I just thought of would be potentially switching the player cards to a table format (similar in style to the results page table or the below image). Would make it less "fun" of a design but could make it more resistant to status changes and easier to show the options available for each user.

![An example table format](https://user-images.githubusercontent.com/24768574/121983578-66246c80-cd46-11eb-9647-e001680960f8.png)

Reply:
That could make all the actions clearer, but I would be worried about the space that takes up - I think just making a hover or dropdown more obvious might be a cleaner solution. However, it doesn't matter much for this PR, so I'll resolve this discussion and we can continue it in the future based on teacher feedback.